Xamarin.Forms.Platform.Android/InnerGestureListener.cs

```diff
@@ -11,21 +11,21 @@ internal class InnerGestureListener : Object, GestureDetector.IOnGestureListener
 	{
 		readonly TapGestureHandler _tapGestureHandler;
 		readonly PanGestureHandler _panGestureHandler;
-        readonly SwipeGestureHandler _swipeGestureHandler;
+        	readonly SwipeGestureHandler _swipeGestureHandler;
 		bool _isScrolling;		
 		float _lastX;
 		float _lastY;
 		bool _disposed;
 
-        Func<float, float, bool> _swipeDelegate;
-        Func<bool> _swipeCompletedDelegate;
+        	Func<float, float, bool> _swipeDelegate;
+        	Func<bool> _swipeCompletedDelegate;
```

Comment:
this is still using spaces

Reply:
Sorry I'm not following - I've used tabs everywhere I'm not sure what changes I'm suppose to make, same with the SwipeDirection one, I've used tabs there too?